### DISCUSSION TITLE
Budibase forms

### DISCUSSION THREAD

**Discussion by massacarlos** [Budibase/budibase#11802] [General]
Dear Sirs: 
An ERP was developed on the Bubble platform, but due to the new pricing we are rewriting the system on another platform. Before delving deeper into the Budibase tool, I would like to know if it is possible to create a form with several pieces of information in one line. See the form created with Bubble.
![Picture_4](https://github.com/Budibase/budibase/assets/102739769/08a6d4bb-c368-48f9-90b3-76ee1325dbb8)
![Picture_3](https://github.com/Budibase/budibase/assets/102739769/0a260c6f-f170-4b12-a1f9-6707bbfb82c2)
![Picture_2](https://github.com/Budibase/budibase/assets/102739769/38780976-8fb3-453a-803a-bb4be0583fda)
![Picture_1](https://github.com/Budibase/budibase/assets/102739769/b80308b3-6222-43b6-996b-b92a7474101a)

**Comment by melohagan** [comment]
HI @massacarlos 

Looking at the screenshots, I can see no reason why Budibase couldn't functionally replicate your screens, although the structure / appearance may obviously be a bit different.
